src/util.ts

```diff
@@ -50,24 +50,22 @@ export function debug(msg: any) {
 
 declare global { interface String { replaceAll(str: string, newStr: string): string; } }
 
+/**
+ * String.prototype.replaceAll() polyfill
+ * https://gomakethings.com/how-to-replace-a-section-of-a-string-with-another-one-with-vanilla-js/
+ * @author Chris Ferdinandi
+ * @license MIT
```

Comment:
It should not be moved to outside, as it is a general polyfill function, not only for replaceAll()

Reply:
But without this, we are missing a tooltip for this function

Reply:
Can add another one if needed.

Definition from the Internet:

> A polyfill is a piece of code used to provide modern functionality on older browsers that do not natively support it

Reply:
I moved it to the place where it was before. Other comments can be added by you or in different PRs